### DISCUSSION TITLE
Cannot receive GitHub Student Developer Pack benefits

### DISCUSSION THREAD

**Discussion by SmokingPuddle58** [community/community#76041] [GitHub Education]
### Hi everyone,

Hi all! I have re-verified my student status on 1 Oct, and since then more than 7 weeks have been passed, and I still cannot access to any of the benefits. I tried to open a ticket for this issue but I received no reply since last week. So in this case what should I do? Thanks 🙏

**Comment by ghost** [comment]
Hello @SmokingPuddle58 

I think i saw your question few days ago, unfortunatelly i have no direct answer for you.

However what i think is that you must contact github support for this problem since if your account is verified as a student and still cant use its benefits probably its not your fault and it requires help from Github support team. [Link to github support](https://support.github.com/).

hope this helps

**Reply by SmokingPuddle58** [reply]
I think I have contacted them for this issue but I received no reply from them for more than one weeks. Anyway I will try to find them again. Thanks!

**Comment by ghost** [comment]
Hello @SmokingPuddle58 

I will suggest you to wait few more days as GitHub support teams takes time to review/resolve issue .

Please re-check that while applying for **GitHub Student Pack** must have done the following things --

- Your **location service** must not be blocked by your Browser. 
- Completed your GitHub account **Billing Information** with your full legal name as it appears on your academic affiliation documentation.
- Verified your **Academic Email Address** on your GitHub Account.
- **2 Step Verification** must be activated.
- Personalized your **public GitHub profile** with your photo, your name, your pronouns, and more.
- Make a good **README** for your public GitHub Profile, customized it with your interests,and publish it for the world to see.

**Reply by SmokingPuddle58** [reply]
I think I have finished all the steps you stated. But for last time I apply I did not finish billing information and did not have README file also, but I still received benefit, so I believe the issue is not related to these.

**Comment by RPR21** [comment]
@SmokingPuddle58 
Hi ! I am also a verified student since October 1 the same as of you but unable to redeem the benefits. Can you please let me know what you have done in case you get the benefits? I am just tired of writing to them and see their automated replies. I wonder if there is any other way to contact the github.

**Reply by SmokingPuddle58** [reply]
I'm also waiting for this 🫠

I think finding GitHub support is the only way but they haven't reply me for more than a week so what I can do is just wait 😔.

~~I wish I can enjoy my benefit before I graduate.~~

**Comment by aprylewu** [comment]
Having same issue. Dont know whether there is a way to contact github? email,for example?
![image](https://github.com/community/community/assets/129849514/3e7d4365-8193-41c1-8b46-8e77cefce8f5)
![image](https://github.com/community/community/assets/129849514/f7205a5f-27f5-4535-8c28-e2393e477ed6)
